chromium_src/net/proxy_resolution/proxy_resolution_service.cc

```diff
@@ -0,0 +1,22 @@
+/* Copyright (c) 2019 The Brave Authors. All rights reserved.
```

Comment:
this stuff is all going away with an the PR that @darkdh and I have been working on, but please review this @darkdh

Reply:
Temporary workaround for the old way looks good to me. I will remove this when I rebase https://github.com/brave/brave-core/pull/2647

Reply:
oh, actually I didn't even realize what was happening here, but we shouldn't be doing this. This method was called internally and just recreating it is not going to work correctly

Reply:
@darkdh is the other tor PR ready because it would be better to rebase this on it?